### PR TITLE
Fix Bug 1440895: Add dj.debug package to requirements-dev

### PR DESCRIPTION
**Pull request by phlax**
This allows the use of a @debug decorator that can be used to
print out the sql used in a method or function.

Also provides a contextmanager to do similar for a block of code.

Ideally, we add this to staging, and then we can check/share performance changes

### CODE REVIEW

File: bin/post_compile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+pip install -r requirements-dev.txt
```
Comment (jotes) [inline bin/post_compile]: I'm not sure if Heroku won't run that in the production environment.

Comment (phlax) [reply bin/post_compile]: good spot!

i wonder how we can do this only on staging

File: bin/post_compile [hunk 2]
```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+
+# Install requirements-dev.txt on staging
+if [ "$HEROKU_APP_NAME" == "mozilla-pontoon-staging" ]; then
```
Comment (mathjazz) [inline bin/post_compile]: Instead of hardcoding the instance name, maybe use `DJANGO_DEV` here and set it to True on stage?

Comment (phlax) [reply bin/post_compile]: hmm, maybe they are independent

```
DJANGO_DEBUG:                 False
DJANGO_DEV:                   False
DJANGO_LOG_LEVEL:             DEBUG
```

ill update and see what happens...

Comment (mathjazz) [reply bin/post_compile]: We might need a new variable - `STAGE`.

Comment (phlax) [reply bin/post_compile]: @mathjazz ive updated with `DJANGO_DEV` and checked staging/logs etc - all looks good.

i think this is the right way to go - but i can change to `STAGE` if you want

**Comment by mathjazz** [comment]
Setting `DJANGO_DEV=True` impacts other things, which I don't think are desirable on stage, especially Django Toolbar:

https://github.com/mozilla/pontoon/blob/master/pontoon/settings/base.py#L52
https://github.com/mozilla/pontoon/blob/master/pontoon/settings/dev.py
https://github.com/mozilla/pontoon/blob/master/pontoon/api/urls.py

I think we should add a `STAGE` env variable.

**Comment by phlax** [comment]
> I think we should add a STAGE env variable.

ah, ok, on it - sorry this has taken a few attempts to get right

**Pull request merged**